core/src/main/kotlin/configuration.kt

```diff
@@ -80,6 +81,7 @@ interface DokkaConfiguration : Serializable {
     val modules: List<DokkaModuleDescription>
     val pluginsClasspath: List<File>
     val pluginsConfiguration: Map<String, String>
+    val globalMainPage: String
```

Comment:
IMHO this should be a `Path` or better yet `File` to be more consistent with our current configuration

Reply:
I am also not 100% happy with the name. Is there a "non-global main page" that somebody who configures this should care? 

Ideas: 
- `mainPage: File?`
- `mainPageFile: File?`

Reply:
Also: I think there should be a way of figuring the "main page" out during generation. I think this can be tackled by me during this bigger ticket for "multi-module documentation". Therefore this configuration _might_ just be temporary 🙄

Reply:
Even if this can be figured out by runner, it still should be configurable. I can imagine cases in which the "main page" is something one level higher than dokka's main page.

Reply:
Can you give me an example, just so I have a better image of it?
In which scenario would we want to configure this different?

Reply:
Library documentation can be part of some bigger website and creators may want this logo to lead to the homepage of the library that can have a high-level explanation with examples and from that links to interesting dokka pages.
I can also think about a case in which there is collection of libraries that have separate documentations but the common homepage.

Reply:
I changed it to URI due to both `File` and `Path` limitations regarding possible schemes